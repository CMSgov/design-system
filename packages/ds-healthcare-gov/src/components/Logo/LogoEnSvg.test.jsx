import LogoEnSvg from './LogoEnSvg';
import React from 'react';
import { shallow } from 'enzyme';

describe('LogoEnSvg', function () {
  it('renders English SVG logo', () => {
    expect(shallow(<LogoEnSvg />)).toMatchSnapshot();
  });
});
