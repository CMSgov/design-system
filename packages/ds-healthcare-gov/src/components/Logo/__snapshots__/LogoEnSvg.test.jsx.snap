// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoEnSvg renders English SVG logo 1`] = `
<svg
  aria-labelledby="hc-c-logo__title"
  height="25"
  viewBox="0 0 253.05 31.65"
  width="200"
  xmlns="http://www.w3.org/2000/svg"
>
  <title
    id="hc-c-logo__title"
  >
    HealthCare.gov
  </title>
  <path
    className="hc-c-logo__health"
    d="M0,.61H4.68V9.75H15.74V.61H20.4V24.85H15.74V14.22H4.67V24.85H0Z"
  />
  <path
    className="hc-c-logo__health"
    d="M41.41,17.18H27a4.8,4.8,0,0,0,1.67,3,5.25,5.25,0,0,0,3.47,1.13,6,6,0,0,0,4.33-1.76l3.79,1.78a8.92,8.92,0,0,1-3.39,3,10.56,10.56,0,0,1-4.69,1,9.25,9.25,0,0,1-6.87-2.66A9.06,9.06,0,0,1,22.62,16,9.35,9.35,0,0,1,25.26,9.2a8.89,8.89,0,0,1,6.63-2.71A9.21,9.21,0,0,1,38.77,9.2a9.78,9.78,0,0,1,2.65,7.16Zm-4.5-3.54a4.51,4.51,0,0,0-1.76-2.44,5.13,5.13,0,0,0-3-.94,5.41,5.41,0,0,0-3.3,1.05,5.88,5.88,0,0,0-1.65,2.32Z"
  />
  <path
    className="hc-c-logo__health"
    d="M57.28,6.92h4.5V24.85h-4.5V23a8.66,8.66,0,0,1-2.64,1.8,7.44,7.44,0,0,1-2.87.55,8,8,0,0,1-6-2.69,9.39,9.39,0,0,1-2.53-6.7,9.61,9.61,0,0,1,2.45-6.8,7.81,7.81,0,0,1,6-2.65,7.56,7.56,0,0,1,3,.61,8.26,8.26,0,0,1,2.59,1.78Zm-4.73,3.69a4.55,4.55,0,0,0-3.46,1.47,5.9,5.9,0,0,0,0,7.6,4.54,4.54,0,0,0,3.46,1.5,4.62,4.62,0,0,0,3.51-1.47,5.4,5.4,0,0,0,1.42-3.86,5.26,5.26,0,0,0-1.4-3.79,4.67,4.67,0,0,0-3.52-1.45Z"
  />
  <path
    className="hc-c-logo__health"
    d="M65,0h4.48V24.85H65Z"
  />
  <path
    className="hc-c-logo__health"
    d="M80.61,21.21a10.17,10.17,0,0,0,3-.48v2.48a6,6,0,0,1-4,1.71c-3.72,0-5.32-2.19-5.32-4.69V10.78H72V6.91h2.31V2.29h4.5V6.91h4.9v3.87h-4.9v7.63C78.79,20,78.93,21.21,80.61,21.21Z"
  />
  <path
    className="hc-c-logo__health"
    d="M86.77,0h4.46V8.73A10.28,10.28,0,0,1,93.88,7a6.78,6.78,0,0,1,2.7-.57,6.06,6.06,0,0,1,4.5,1.85A6.47,6.47,0,0,1,102.64,13V24.85H98.21V17a19.9,19.9,0,0,0-.3-4.22,2.93,2.93,0,0,0-1-1.65,2.87,2.87,0,0,0-1.77-.54,3.31,3.31,0,0,0-2.35.91A4.84,4.84,0,0,0,91.43,14a19.82,19.82,0,0,0-.2,3.69v7.2H86.77Z"
  />
  <path
    className="hc-c-logo__care"
    d="M129.77,4.85l-3.23,3.07a10,10,0,0,0-7.4-3.48,8.11,8.11,0,0,0-8.23,8v.24a8.61,8.61,0,0,0,1,4.3,7.47,7.47,0,0,0,3,3A8.64,8.64,0,0,0,119.24,21a8.77,8.77,0,0,0,3.67-.75,13,13,0,0,0,3.65-2.73l3.13,3.26a17.06,17.06,0,0,1-5.07,3.63,13.73,13.73,0,0,1-5.45,1,12.56,12.56,0,0,1-9.25-3.58,12.38,12.38,0,0,1-3.6-9.18A12.63,12.63,0,0,1,108,6.25a12.18,12.18,0,0,1,4.7-4.53A13.32,13.32,0,0,1,119.24,0,14,14,0,0,1,129.77,4.85Z"
  />
  <path
    className="hc-c-logo__care"
    d="M145,6.92h4.49V24.85H145V23a8.66,8.66,0,0,1-2.64,1.8,7.44,7.44,0,0,1-2.87.55,8,8,0,0,1-6-2.69,9.39,9.39,0,0,1-2.53-6.7,9.61,9.61,0,0,1,2.45-6.8,7.81,7.81,0,0,1,6-2.65,7.56,7.56,0,0,1,3,.61,8.37,8.37,0,0,1,2.62,1.83Zm-4.73,3.69a4.55,4.55,0,0,0-3.46,1.47,5.33,5.33,0,0,0-1.37,3.77,5.37,5.37,0,0,0,1.4,3.82,4.54,4.54,0,0,0,3.46,1.5,4.65,4.65,0,0,0,3.51-1.47,5.42,5.42,0,0,0,1.39-3.86,5.25,5.25,0,0,0-1.39-3.79,4.69,4.69,0,0,0-3.53-1.44Z"
  />
  <path
    className="hc-c-logo__care"
    d="M152.7,6.92h3.86v3.79a8.62,8.62,0,0,1,2.91-3.45c2.23-1.32,4.59.47,4.59.47v3.74a5.49,5.49,0,0,0-4.58.38c-1.54,1-2.31,2.71-2.31,5.21v7.79H152.7Z"
  />
  <path
    className="hc-c-logo__care"
    d="M183.82,17.18H169.38a4.8,4.8,0,0,0,1.67,3,5.21,5.21,0,0,0,3.43,1.17,6,6,0,0,0,4.33-1.76l3.79,1.78a8.92,8.92,0,0,1-3.39,3,10.6,10.6,0,0,1-4.7,1,9.25,9.25,0,0,1-6.87-2.66A9.06,9.06,0,0,1,165,16.05a9.34,9.34,0,0,1,2.65-6.81,8.89,8.89,0,0,1,6.63-2.71,9.21,9.21,0,0,1,6.89,2.71,9.82,9.82,0,0,1,2.65,7.16Zm-4.5-3.54a4.51,4.51,0,0,0-1.76-2.44,5.13,5.13,0,0,0-3-.94,5.41,5.41,0,0,0-3.3,1.05,5.88,5.88,0,0,0-1.65,2.32Z"
  />
  <path
    className="hc-c-logo__gov"
    d="M189.37,25.08a1.73,1.73,0,0,1-1.29-.54,1.85,1.85,0,0,1,0-2.61,1.82,1.82,0,0,1,2.57,0h0a1.85,1.85,0,0,1,0,2.61,1.73,1.73,0,0,1-1.29.54Z"
  />
  <path
    className="hc-c-logo__gov"
    d="M204.74,31.64a12.27,12.27,0,0,1-4.08-.64,8.12,8.12,0,0,1-2.92-1.71,10.05,10.05,0,0,1-2-2.87h2.07A6.88,6.88,0,0,0,200.52,29a8.79,8.79,0,0,0,4.1.88,8.6,8.6,0,0,0,4.08-.9,5.58,5.58,0,0,0,2.4-2.29,9.94,9.94,0,0,0,.72-4.38V20.85l-.36.45a8.51,8.51,0,0,1-3.11,2.47,9.26,9.26,0,0,1-8.46-.34,8.51,8.51,0,0,1-3.29-3.23,8.94,8.94,0,0,1-1.16-4.46,9.13,9.13,0,0,1,9.07-9.09,8.39,8.39,0,0,1,3.66.81,10.31,10.31,0,0,1,3.29,2.69l.35.41V7.11h1.91V21.19a17,17,0,0,1-.65,5.45A7,7,0,0,1,210,30.35a10.24,10.24,0,0,1-5.28,1.3Zm.08-23.18a7.52,7.52,0,0,0-3.73,1,7.07,7.07,0,0,0-2.71,2.69,7.45,7.45,0,0,0-1,3.71,6.75,6.75,0,0,0,2,5,8,8,0,0,0,10.51,0,7,7,0,0,0,2-5.16A7.51,7.51,0,0,0,211,12a6.72,6.72,0,0,0-2.59-2.59,7.24,7.24,0,0,0-3.61-1Z"
  />
  <path
    className="hc-c-logo__gov"
    d="M226.65,25.08a8.66,8.66,0,0,1-6.6-2.74,9.41,9.41,0,0,1-.14-12.75,9.16,9.16,0,0,1,12.94-.51c.18.17.35.33.51.51a9.41,9.41,0,0,1-.14,12.77,8.6,8.6,0,0,1-6.58,2.74Zm0-16.58a6.86,6.86,0,0,0-5.09,2.19A7.34,7.34,0,0,0,219.44,16a7.56,7.56,0,0,0,1,3.72,6.93,6.93,0,0,0,2.62,2.67,7.41,7.41,0,0,0,7.24,0,6.93,6.93,0,0,0,2.62-2.67,7.56,7.56,0,0,0,1-3.72,7.33,7.33,0,0,0-2.13-5.28,6.87,6.87,0,0,0-5.07-2.19Z"
  />
  <polygon
    className="hc-c-logo__gov"
    points="245.03 24.61 244.86 24.61 236.79 7.09 238.8 7.09 244.76 20.04 244.95 20.43 245.13 20.04 251.02 7.09 253.05 7.09 245.03 24.61 245.03 24.61"
  />
</svg>
`;
